src/Operators/vorticity_operators.jl

```diff
@@ -3,3 +3,31 @@
 
 """ Vertical vorticity associated with horizontal velocities u, v. """
 @inline ζ₃ᶠᶠᵃ(i, j, k, grid, u, v) = Γᶠᶠᵃ(i, j, k, grid, u, v) / Azᶠᶠᵃ(i, j, k, grid)
+
+#####
+##### Vertical circulation at the corners of the cubed sphere needs to treated in a special manner.
+##### See: https://github.com/CliMA/Oceananigans.jl/issues/1584
+#####
+
+@inline function Γᶠᶠᵃ(i, j, k, grid::ConformalCubedSphereFaceGrid, u, v)
+    # South-west corner
+    if i == 1 && j == 1
+        return Δy_vᶜᶠᵃ(i, j, k, grid, v) - δyᵃᶠᵃ(i, j, k, grid, Δx_uᶠᶜᵃ, u)
```

Comment:
@ali-ramadhan and @glwagner I find the δyᵃᶠᵃ slightly distracting. I think of δyᵃᶠᵃ as a difference operator. Its a bit subjective but I might have written this just using Δy_vᶜᶠᵃ and Δx_uᶠᶜᵃ notation e.g

-Δx_uᶠᶜᵃ(i, j, k, grid) + Δx_uᶠᶜᵃ(i, j-1, k, grid)


so it felt more like a line integral?

Reply:
Ah I was just going with equation (13) of the 2004 cubed sphere paper 

![image](https://user-images.githubusercontent.com/20099589/114763756-2b478d80-9d31-11eb-8eb8-d0de89f04589.png)

but for these corner cases we're already mixing the two so we could switch to just using `Δx_uᶠᶜᵃ` and `Δy_vᶜᶠᵃ` for clarity?

Reply:
That's a good point! I always thought that notation was a perverse way to write a line integral though 😄 .

Reply:
Hmm, I agree that the delta notation obfuscates the line integral. Like @ali-ramadhan I was simply transcribing the paper's notation, I think what @christophernhill suggests is a big improvement.